src/main/kotlin/probation/Reporting.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.hmpps.architecture.probation
+
+import com.structurizr.model.Model
+import com.structurizr.model.Person
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.HMPPSSoftwareSystem
+import uk.gov.justice.hmpps.architecture.shared.Tags
+
+class Reporting private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var ndmis: SoftwareSystem
+
+    lateinit var dataInnovation: Person
+    lateinit var nart: Person
+
+    lateinit var npsPerformance: Person
+    lateinit var prisonPerformance: Person
+    lateinit var communityPerformance: Person
+    lateinit var crcPerformance: Person
```

Comment:
<nitpick> would these variables benefit from "Team", "Analyst", etc matching their string names? `npsPerformanceOfficer` really screams that it's an actor :).

Reply:
Yeah, you're right. I was rushing too much. ⌛  changes incoming

Reply:
Done in 7afc0c712c5ae01797f0bd7f04652b2af14c42f2

Reply:
Huh, I don't actually use these actors outside the object.
But if I make them private, do we need a way to discover that these are already defined? Maybe I'm overthinking?

Reply:
I've been having the same thoughts. I'm *almost* tempted to suggest we define people completely separately, to promote reuse, but that'll make adding a new system a bit more onerous (3+ files to edit rather than 2) in the short term.

It might make sense to make them private for now. That would force us to at least think about refactoring them to somewhere more appropriate if we need to reference them elsewhere. That could be a different PR though, it doesn't block this :).